src/services/api.js

```diff
@@ -1,14 +1,85 @@
 const BASE_URL = "https://bootcamp-api.codeit.kr";
 
-export async function getCards() {
-  const response = await fetch(`${BASE_URL}/api/sample/folder`);
-  const result = await response.json();
-  return result;
+export async function getSampleCards() {
+  try {
+    const response = await fetch(`${BASE_URL}/api/sample/folder`);
+    const result = await response.json();
+    if (response.ok) return result;
+    else throw new Error(result.message || "Failed to fetch sample cards");
+  } catch (error) {
+    console.error("Error fetching sample cards", error);
+    throw error;
+  }
+}
+
+export async function getSampleUser() {
+  try {
+    const response = await fetch(`${BASE_URL}/api/sample/user`);
+    const result = await response.json();
+    if (response.ok) return result;
+    else
+      throw new Error(
+        result.message || "Failed to fetch sample user information"
+      );
+  } catch (error) {
+    console.error("Error fetching sample user information", error);
+    throw error;
+  }
+}
+
+export async function getUser() {
+  try {
+    const response = await fetch(`${BASE_URL}/api/users/4`);
+    const result = await response.json();
+    if (response.ok) return result;
+    else throw new Error(result.message || "Failed to fetch user information");
+  } catch (error) {
+    console.error("Error fetching user information", error);
+    throw error;
+  }
 }
 
-export async function getUsers() {
-  const response = await fetch(`${BASE_URL}/api/sample/user`);
-  const result = await response.json();
+export async function getFolders() {
+  try {
+    const response = await fetch(`${BASE_URL}/api/users/4/folders`);
+    const result = await response.json();
+    if (response.ok) return result;
+    else
+      throw new Error(result.message || "Failed to fetch folder information");
+  } catch (error) {
+    console.error("Error fetching folder information", error);
+    throw error;
+  }
+}
+
+export async function getCards() {
+  try {
+    const response = await fetch(`${BASE_URL}/api/users/4/links`);
+    const result = await response.json();
+    if (response.ok) return result;
+    else
+      throw new Error(
+        result.message || `Failed to fetch user's card information`
```

Comment:
```suggestion
        result.message || "Failed to fetch user's card information"
```
문자열에 템플릿리터럴로 사용되지 않는다면 백틱(`) 대신 따옴표를 사용해주세요!
백틱을 사용하면 템플릿 리터럴이 올거라고 읽는이로하여금 기대하게 하거든요ㅎㅎ